ogcapi-features/README.md

```diff
@@ -157,10 +157,11 @@ The following Link relations must exist in the Item object returned from the `/c
 | **rel**  | **href**                                     | **Media Type**       | **From**            | **Description**                                     |
 | -------- | -------------------------------------------- | -------------------- | ------------------- | --------------------------------------------------- |
 | `root`   | `/`                                          | application/json     | STAC API - Features | The root URI                                        |
+| `collection` | `/collections/{collectionId}`            | application/json     | OAFeat              | The containing Collection |
 | `parent` | `/collections/{collectionId}`                | application/json     | OAFeat              | Parent reference, usually the containing Collection |
 | `self`   | `/collections/{collectionId}/items/{itemId}` | application/geo+json | OAFeat              | Self reference                                      |
 
-Note that the `parent` link for an Item should point to the containing Collection
+Note that the `parent` and `collection` link for an Item should point to the containing Collection
```

Comment:
The original text for this is overly-constrained, so we should fix that here. 

This statement is true for the collection link, but the parent link may either point to the collection or to a catalog, so there needs to be a qualifier like "if the parent link is to the collection, make sure it po;ints to the collection rather than the `/items`"

It would also be good to clarify here that the parent link may be to either a collection (typical) or to a sub-catalog URI (currently unspecified by STAC API).

Reply:
The latest commit updates this language.  Let me know if that looks ok.